packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -65,3 +65,13 @@ export const FUTURE_FLAGS = {
   // We will advertise this flag when the feature is complete
   // [NEW_STYLE_STACK_SYNTHESIS]: 'true',
 };
+
+const FUTURE_FLAGS_DEFAULTS: { [key: string]: boolean } = {
```

Comment:
Shouldn't this map have all the feature flags? I am missing e.g. NEW_STYLE_STACK_SYNTHESIS

Reply:
Currently, it's not opted in by default on new project, so it shouldn't matter that much. When it's added to the `FUTURE_FLAGS` section above, a unit test will prompt for it to be added to the defaults as well.

I can add it now anyway if you feel so.